app/controllers/api/v1/passwords_controller.rb

```diff
@@ -4,6 +4,36 @@ class PasswordsController < DeviseTokenAuth::PasswordsController
       protect_from_forgery with: :exception, unless: :json_request?
       include Api::Concerns::ActAsApiRequest
       skip_before_action :check_json_request, on: :edit
+
+      def edit
```

Comment:
can you explain why do we need to override the gem action now? maybe point to some links and documentation